src/commands/link/index.ts

```diff
@@ -1,28 +1,40 @@
 import chalk from 'chalk';
 import path from 'path';
 import untildify from 'untildify';
-import { buildSpecFromPath } from '../';
-import BaseCommand from '../base-command';
+import { buildSpecFromPath } from '../..';
+import BaseCommand from '../../base-command';
+import BaseTable from '../../base-table';
+
 declare const process: NodeJS.Process;
 
 export default class Link extends BaseCommand {
   async auth_required(): Promise<boolean> {
     return false;
   }
 
-  static description = 'Link a local component to the host to be used to power local deployments.';
+  static description = 'Link a local component to the host to be used to power local deployments or list all linked components.';
```

Comment:
The descriptions in this file should be changed to only be relevant to the actual linking action, and a description should be added to the `link.ts` file to describe the listing action